src/AppBundle/Controller/TicketsController.php

```diff
@@ -30,6 +31,11 @@ public function getAction(Ticket $ticket)
     public function freeAction(Ticket $ticket)
     {
         $em = $this->getDoctrine()->getManager();
+
+        if ($ticket->getStatus() === Ticket::STATUS_FREE) {
+            throw new TicketStatusConflictException('Ticket is already free');
```

Comment:
@spolischook , remove check and exception like this from method ticket/reserve too or not?